docs/docs-content/clusters/edge/local-ui/cluster-management/local-to-central.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_label: "Pair Local Cluster with Palette"
+title: "Pair Local Cluster with Palette"
+description:
+  "Learn how to move a local cluster created on hosts with airgap installation mode to be managed by a Palette instance."
+hide_table_of_contents: false
+sidebar_position: 55
+tags: ["edge"]
+---
+
+Clusters provisioned on hosts that are installed in the `airgap` installation mode are managed locally by Local UI. You
```

Comment:
```suggestion
Clusters provisioned on hosts that are installed in the airgap installation mode are managed locally by Local UI. You
```
It feels a bit odd to have codeblocks here for airgap. I know it's a UI option, so I wonder if maybe italics is the better option, as we are using it as a term.